### PR TITLE
feat( Integration test):  edit params in details section

### PR DESCRIPTION
**Pull request by abhinandan13jan**
## Fixes 

https://issues.redhat.com/browse/HAC-4840

## Description

Add Edit Params modal and params in ITS details view

## Type of change

- [x] Feature
- [ ] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Screen shots / Gifs for design review 

Screenshot 2023-09-27 at 8 24 23 PM
Screenshot 2023-09-27 at 8 27 46 PM

Screenshot 2023-10-17 at 5 42 01 PM
Screenshot 2023-10-17 at 5 42 14 PM

## Browser conformance: 

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Review by rohitkrai03 — COMMENTED** [review]
@abhinandan13jan I think we need to increase the width of the modal for this use case. It seems very narrow to me. @MariaLeonova what do you think?

**Comment by MariaLeonova** [comment]
This is what I originally had for **Edit parameters** modal:
image
I would suggest the width of the params modal to be at least 872px, which would be consistent with **Create new build secret** modal on import.

**Comment by abhinandan13jan** [comment]
Updated @rohitkrai03 @MariaLeonova Using medium variant like create secrets as suggested. Large would be too wide

**Comment by MariaLeonova** [comment]
looks good, @abhinandan13jan !

File: src/components/IntegrationTest/tabs/IntegrationTestOverviewTab.tsx [hunk 1]
```diff
@@ -124,6 +134,36 @@ const IntegrationTestOverviewTab: React.FC<IntegrationTestOverviewTabProps> = ({
...
+              {params && (
+                <DescriptionListGroup data-test="its-overview-params">
+                  <DescriptionListTerm>
+                    Parameters{' '}
+                    <Tooltip content="Parameters which will be added to the integration Tekton pipeline run.">
+                      <OutlinedQuestionCircleIcon />
+                    </Tooltip>
+                  </DescriptionListTerm>
+                  <DescriptionListDescription>
+                    {`${params.length} parameters`}
```
Comment (karthikjeeyar) [inline src/components/IntegrationTest/tabs/IntegrationTestOverviewTab.tsx:146]: use pluralized string only when the length is greater than `1`.
image

File: src/components/IntegrationTest/EditParamsModal.tsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+import FormikParamsField from './FormikParamsField';
+
+type EditParamsModalProps = ComponentProps & {
+  intTest: IntegrationTestScenarioKind;
+};
+
+export const EditParamsModal: React.FC<EditParamsModalProps> = ({ intTest, onClose }) => {
+  const [error, setError] = React.useState<string>();
+
+  const getFormParamValues = (params) => {
```
Comment (karthikjeeyar) [inline src/components/IntegrationTest/EditParamsModal.tsx:30]: add type for params.

**Review by karthikjeeyar — COMMENTED** [review]
Integration test name is not outlined in the design doc. I am not sure if this is required here or not because it already showing up in the background too, @MariaLeonova thoughts?


image

**Comment by abhinandan13jan** [comment]
> Integration test name is not outlined in the design doc. I am not sure if this is required here or not because it already showing up in the background too, @MariaLeonova thoughts?
> 
> image

removed

Screenshot 2023-10-17 at 5 42 01 PM

**Pull request merged**